docs/sources/auth/generic-oauth.md

```diff
@@ -59,6 +59,10 @@ Check for the presence of a role using the [JMESPath](http://jmespath.org/exampl
 
 See [JMESPath examples](#jmespath-examples) for more information.
 
+> Only available in Grafana v7.2+.
+
+Customize user login using `login_attribute_path` configuration option. Grafana will attempt to evaluate the `login_attribute_path` JMESPath expression against the ID token first and then, if no value found, against the HTTP response obtained from querying the UserInfo endpoint specified via the `api_url` configuration option. The attribute name which will be used for extracting ID token from the returned OAuth token can be customized with `id_token_attribute_name` option.
```

Comment:
```suggestion
Customize user login using `login_attribute_path` configuration option. Grafana will attempt to evaluate the `login_attribute_path` JMESPath expression against the ID token first and then, if no value is found, against the HTTP response obtained by querying the UserInfo endpoint specified via the `api_url` configuration option. The attribute name which will be used for extracting the ID token from the returned OAuth token can be customized with the `id_token_attribute_name` option.
```

Reply:
Best practice is to keep sentences at 25 words or less. Please break up the second sentence.

Also, try rewriting this in present tense. "Grafana attempts" instead of "Grafana will attempt" and so on.

Reply:
@oddlittlebird could you take a quick look at the changes?